jsf/subsystem/src/main/java/org/jboss/as/jsf/subsystem/JSFResourceDefinition.java

```diff
@@ -59,11 +65,27 @@ public class JSFResourceDefinition extends PersistentResourceDefinition {
                     .setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES)
                     .build();
 
+    /*
+    private MailServerDefinition(final PathElement path, AttributeDefinition[] attributes) {
+        super(new Parameters(path,
+                MailExtension.getResourceDescriptionResolver(MailSubsystemModel.MAIL_SESSION, MailSubsystemModel.SERVER_TYPE))
+                .setAddHandler(new MailServerAdd(attributes))
+                .setRemoveHandler(new MailServerRemove(attributes))
+                .setCapabilities(SERVER_CAPABILITY)
+        );
+        this.attributes = Arrays.asList(attributes);
+    }
+     */
```

Comment:
I assume this was the example and can be removed :)

Reply:
Good grief. Yes, it can. :P